src/LoginFormContainer.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import LoginForm from './LoginForm';
+
+import {
+  requestLogin,
+  changeLoginField,
+  setAccessToken,
+} from './actions';
+
+import { get } from './utils';
+
+export default function LoginFormContainer() {
+  const loginFields = useSelector(get('loginFields'));
+  const isLoggedIn = useSelector(get('accessToken')) !== null;
+
+  const dispatch = useDispatch();
+
+  const handleSubmit = () => {
+    dispatch(requestLogin());
+  };
+
+  const handleChange = ({ name, value }) => {
+    dispatch(changeLoginField({ name, value }));
+  };
+
+  const handleClickLogOut = () => {
+    dispatch(setAccessToken(null));
+  };
+
+  if (isLoggedIn) {
+    return (
+      <button type="button" onClick={handleClickLogOut}>
+        Log out
+      </button>
+    );
+  }
+
+  return (
+    <LoginForm
+      fields={loginFields}
+      onSubmit={handleSubmit}
+      onChange={handleChange}
+    />
+  );
```

Comment:
```suggestion
  return (
    <>
      {isLoggedIn ? (
        <button type="button" onClick={handleClickLogOut}>
          Log out
        </button>
      ) : (
        <LoginForm
          fields={loginFields}
          onSubmit={handleSubmit}
          onChange={handleChange}
        />
      )}
    </>
  );
```

지금도 잘해주셨지만, 이렇게 처리해주는건 어떤가요? 개인적으로는 조금 더 명확할 것 같아요~